.pylintrc

```diff
@@ -28,7 +28,7 @@ disable=
 
 [FORMAT]
 max-line-length=125
-max-module-lines=1000
+max-module-lines=1100
```

Comment:
This seems like a good time to refactor basebackup.py into (at least) two or three files :grin: From the top of my head I would probably separate the logic along "regular" and delta basebackup functionality by Mixin to the `PGBaseBackup` class, and separate out utilities.

Reply:
Well, I can not agree more, that it needs refactoring, but this ticket took me quite some time (including the delta backups themselves), so I would not like to spend time here right now, considering priorities.